airflow/include/airflow3/composeyml.go.tmpl

```diff
@@ -0,0 +1,163 @@
+x-common-env-vars: &common-env-vars
+  AIRFLOW__API__BASE_URL: "http://localhost:8080"
+  AIRFLOW__API__PORT: 8080
+  AIRFLOW__CORE__AUTH_MANAGER: airflow.api_fastapi.auth.managers.simple.simple_auth_manager.SimpleAuthManager
+  AIRFLOW__CORE__SIMPLE_AUTH_MANAGER_ALL_ADMINS: "True"
+  AIRFLOW__CORE__EXECUTION_API_SERVER_URL: "http://api-server:8080/execution/"
+  AIRFLOW__CORE__EXECUTOR: LocalExecutor
+  AIRFLOW__CORE__LOAD_EXAMPLES: "False"
+  AIRFLOW__CORE__SQL_ALCHEMY_CONN: postgresql://{{ .PostgresUser }}:{{ .PostgresPassword }}@{{ .PostgresHost }}:5432
+  AIRFLOW__DATABASE__SQL_ALCHEMY_CONN: postgresql://{{ .PostgresUser }}:{{ .PostgresPassword }}@{{ .PostgresHost }}:5432
+  AIRFLOW__SCHEDULER__STANDALONE_DAG_PROCESSOR: True
+  AIRFLOW__WEBSERVER__SECRET_KEY: "{{ .ProjectName }}"
+  AIRFLOW__WEBSERVER__RBAC: "True"
+  AIRFLOW__WEBSERVER__EXPOSE_CONFIG: "True"
+  ASTRONOMER_ENVIRONMENT: local
+
+networks:
+  airflow:
+    driver: bridge
+
+volumes:
+  postgres_data:
+    driver: local
+  airflow_logs:
+    driver: local
+
+services:
+  postgres:
+    image: {{ .PostgresRepository }}:{{ .PostgresTag }}
+    restart: unless-stopped
+    networks:
+      - airflow
+    labels:
+      io.astronomer.docker: "true"
+      io.astronomer.docker.cli: "true"
+    ports:
+      {{- if not .AirflowExposePort }}
+      - 127.0.0.1:{{ .PostgresPort }}:5432
+      {{- else }}
+      - {{ .PostgresPort }}:5432
+      {{- end }}
+    volumes:
+      {{- if .DuplicateImageVolumes }}
+      - postgres_data:/var/lib/postgresql/data
+      {{- else }}
+      []
+      {{- end }}
+    environment:
+      POSTGRES_USER: {{ .PostgresUser }}
+      POSTGRES_PASSWORD: {{ .PostgresPassword }}
+
+  db-migration:
+    depends_on:
+      - postgres
+    image: {{ .ProjectName }}/airflow:latest
+    command:
+      - airflow
+      - db
+      - migrate
+    networks:
+      - airflow
+    labels:
+      io.astronomer.docker: "true"
+      io.astronomer.docker.cli: "true"
+      io.astronomer.docker.component: "airflow-db-migration"
+    environment: *common-env-vars
+
+  scheduler:
+    depends_on:
+      - db-migration
+    image: {{ .ProjectName }}/airflow:latest
+    command:
+      - airflow
+      - scheduler
+    restart: unless-stopped
+    networks:
+      - airflow
+    user: astro
+    labels:
+      io.astronomer.docker: "true"
+      io.astronomer.docker.cli: "true"
+      io.astronomer.docker.component: "airflow-scheduler"
+    environment: *common-env-vars
+    volumes:
+      - {{ .AirflowHome }}/dags:/usr/local/airflow/dags:z
+      - {{ .AirflowHome }}/plugins:/usr/local/airflow/plugins:z
+      - {{ .AirflowHome }}/include:/usr/local/airflow/include:z
+      - {{ .AirflowHome }}/tests:/usr/local/airflow/tests:z
+      - airflow_logs:/usr/local/airflow/logs
```

Comment:
we would have to probably continue to conditionally add logs and Postgres data volume using `{{if .DuplicateImageVolumes}}` to ensure it continues to function for Podman 4.x or earlier